Makefile

```diff
@@ -1,34 +1,52 @@
-DOCKER_REPO ?= gcr.io/online-bridge-hackathon-2020
+# Project / Org
+GCP_PROJECT ?= globalbridge-app
+GKE_ZONE ?= europe-west4-b
+
+# Service / App
+GKE_CLUSTER_NAME ?= prod-cluster
+RELEASE_NAME ?= deal
+K8S_NS ?= prod-${RELEASE_NAME}
+EXTERNAL_ADDRESS ?= ${RELEASE_NAME}.prod.globalbridge.app
+
+# Docker Config
+DOCKER_REPO ?= gcr.io/${GCP_PROJECT}
 VERSION ?= $(shell cat VERSION)
-DOCKER_TAG=${DOCKER_REPO}/deal-api:${VERSION}
+DOCKER_TAG = ${DOCKER_REPO}/${RELEASE_NAME}:${VERSION}
 
-EXTERNAL_ADDRES ?= deal.hackathon.globalbridge.app
 
-DDS_K8S_NS ?= deal-api
-GCP_PROJECT ?= online-bridge-hackathon-2020
-GKE_CLUSTER_NAME ?= hackathon-cluster
-GKE_ZONE ?= europe-west3-b
+ifeq (${SILENT},1)
+	VERBOSE_TEST :=
+else
+	VERBOSE_TEST := -v
+endif
+
 
 release: build push
 
-build:
+libdds/.git:
+	git submodule update --init
+
+build: 
 	docker build -t ${DOCKER_TAG} .
 
 push:
 	docker push ${DOCKER_TAG}
 
 deploy: set_gcp_context ensure_ns
-	helm upgrade --install deal-api ./chart \
+	helm upgrade --install ${RELEASE_NAME} ./chart \
 		--set image="${DOCKER_TAG}" \
-		--set externalHostname="${EXTERNAL_ADDRES}" \
-		--namespace ${DDS_K8S_NS} \
+		--set externalHostname="${EXTERNAL_ADDRESS}" \
+		--namespace ${K8S_NS} \
 		--history-max=10
 
 uninstall: set_gcp_context
-	helm del deal-api --namespace ${DDS_K8S_NS}
+	helm del ${RELEASE_NAME} --namespace ${K8S_NS}
```

Comment:
I noticed helm doesn't offer any confirmation. That makes me think delete doesn't belong to makefile. Idea is to reduce risk that we accidentally remove the production version.

Reply:
I agree, will figure out an a check with the user in the Makefile

Reply:
Done. See - https://github.com/online-bridge-hackathon/Deal/pull/3/commits/571867ad7277eb4bc65dcc29b6158240193b6dcd